modules/apps/dispatch/dispatch-test/src/testFunctional/tests/TalendJobScheduler.testcase

```diff
@@ -0,0 +1,69 @@
+@component-name = "portal-dispatch"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Job Scheduler";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given create a Job Scheduler with talend dispatchTaskExecutorType") {
```

Comment:
there's only one test in this file for now, so it could go directly in the test scope